packages/device/src/components/register.component.tsx

```diff
@@ -1,11 +1,65 @@
+import { createSignal, onMount } from 'solid-js';
+import * as QRCode from 'qrcode';
 import { Device } from '../classes';
-import { PincodeComponent } from './pincode.component';
```

Comment:
I think it was useful to have a separate component just for the pincode part. Also as this component is removed from here it would not be used in any other place I think.